### DISCUSSION TITLE
Twitch

### DISCUSSION THREAD

**Discussion by mustafa-archived** [catppuccin/catppuccin#1885] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the requested port be themed?

with the browser extension 'stylus'

### Has the requested port been themed already?

No

### Any additional comments?

Currently working on it https://github.com/mustafakhalaf-git/twitch

**Comment by mustafa-archived** [comment]
@sgoudham alright, It seems complete now

**Comment by sgoudham** [comment]
Transferred over to #1886
